ts/session/sending/MessageSender.ts

```diff
@@ -52,6 +53,57 @@ export async function send(
   );
 }
 
+export async function sendToMediumGroup(
+  message: RawMessage,
+  groupId: string,
+  attempts: number = 3
+): Promise<void> {
+  if (!canSendToSnode()) {
+    throw new Error('lokiMessageAPI is not initialized.');
+  }
+
+  const { plainTextBuffer, timestamp, ttl } = message;
+
+  const ourKey = window.textsecure.storage.user.getNumber();
+
+  const { ciphertext, keyIdx } = await encryptWithSenderKey(
+    plainTextBuffer,
+    groupId,
```

Comment:
what's the difference between `groupId` and `message.destination`?

Reply:
They are the same. Changed to use `message.device` and left a comment that it is actually groupId in that context.